libafl/src/schedulers/testcase_score.rs

```diff
@@ -305,7 +305,12 @@ where
         }
 
         weight *= avg_exec_us / q_exec_us;
-        weight *= libm::log2(q_bitmap_size).max(1.0) / avg_bitmap_size;
+        weight *= if avg_bitmap_size.is_zero() {
```

Comment:
isn't this the same?

Reply:
Not exactly. When `avg_bitmap_size` is zero, `weight` will be `NaN`, which further fails a later assertion.

Reply:
ok i see